public/app/plugins/datasource/prometheus/querybuilder/shared/OperationList.tsx

```diff
@@ -132,6 +134,11 @@ export function OperationList<T extends QueryWithOperations>({
           )}
         </div>
       </Stack>
+      {showConflictMessage && (
+        <p className={styles.hasConflict}>
+          You have conflicting operations in your query, this will result in no data.
+        </p>
+      )}
```

Comment:
I'd keep this message and all logic within OperationEditor. Maybe you can use some error tooltip to display message (similarly as we have it for errors in label)